src/main/java/core/basesyntax/dao/UserDaoImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.List;
+
+public class UserDaoImpl implements UserDao {
+
+    @Override
+    public void add(User user) {
+        Storage.users.add(user);
+    }
+
+    @Override
+    public void checkAge(User user) {
```

Comment:
Not sure this method is connected to our DAO layer

Reply:
agree
move this method out please
it is not connected to our storage

in your dao there should be only simple methods directly manipulating storage

Reply:
delete